client/me/purchases/remove-purchase/index.jsx

```diff
@@ -176,28 +355,273 @@ const RemovePurchase = React.createClass( {
+					value="couldNotInstall"
+					checked={ 'couldNotInstall' === this.state.questionOneRadio }
+					onChange={ this.handleRadioOne } />
+				<span>{ this.translate( 'I couldn\'t install a plugin/theme I wanted.' ) }</span>
+				{ 'couldNotInstall' === this.state.questionOneRadio ? couldNotInstallInput : null }
+			</FormLabel>
+		);
+
+		const tooHardInput = (
+			<FormTextInput
+				className="remove-purchase__reason-input"
+				name="tooHardInput"
+				id="tooHardInput"
+				value={ this.state.questionOneText }
+				onChange={ this.handleTextOne }
+				placeholder={ this.translate( 'Where did you run into problems?' ) } />
+		);
+		reasons.tooHard = (
+			<FormLabel key="tooHard">
+				<FormRadio
+					name="tooHard"
+					value="tooHard"
+					checked={ 'tooHard' === this.state.questionOneRadio }
+					onChange={ this.handleRadioOne } />
+				<span>{ this.translate( 'It was too hard to set up my site.' ) }</span>
+				{ 'tooHard' === this.state.questionOneRadio ? tooHardInput : null }
+			</FormLabel>
+		);
+
+		const didNotIncludeInput = (
+			<FormTextInput
+				className="remove-purchase__reason-input"
+				name="didNotIncludeInput"
+				id="didNotIncludeInput"
+				value={ this.state.questionOneText }
+				onChange={ this.handleTextOne }
+				placeholder={ this.translate( 'What are we missing that you need?' ) } />
+		);
+		reasons.didNotInclude = (
+			<FormLabel key="didNotInclude">
+				<FormRadio
+					name="didNotInclude"
+					value="didNotInclude"
+					checked={ 'didNotInclude' === this.state.questionOneRadio }
+					onChange={ this.handleRadioOne } />
+				<span>{ this.translate( 'This upgrade didn\'t include what I needed.' ) }</span>
+				{ 'didNotInclude' === this.state.questionOneRadio ? didNotIncludeInput : null }
+			</FormLabel>
+		);
+
+		const onlyNeedFreeInput = (
+			<FormTextInput
+				className="remove-purchase__reason-input"
+				name="onlyNeedFreeInput"
+				id="onlyNeedFreeInput"
+				value={ this.state.questionOneText }
+				onChange={ this.handleTextOne }
+				placeholder={ this.translate( 'How can we improve our upgrades?' ) } />
+		);
+		reasons.onlyNeedFree = (
+			<FormLabel key="onlyNeedFree">
+				<FormRadio
+					name="onlyNeedFree"
+					value="onlyNeedFree"
+					checked={ 'onlyNeedFree' === this.state.questionOneRadio }
+					onChange={ this.handleRadioOne } />
+				<span>{ this.translate( 'All I need is the free plan.' ) }</span>
+				{ 'onlyNeedFree' === this.state.questionOneRadio ? onlyNeedFreeInput : null }
+			</FormLabel>
+		);
+
+		const anotherReasonOneInput = (
+			<FormTextInput
+				className="remove-purchase__reason-input"
+				name="anotherReasonOneInput"
+				value={ this.state.questionOneText }
+				onChange={ this.handleTextOne }
+				id="anotherReasonOneInput" />
+		);
+		reasons.anotherReasonOne = (
+			<FormLabel key="anotherReasonOne">
+				<FormRadio
+					name="anotherReasonOne"
+					value="anotherReasonOne"
+					checked={ 'anotherReasonOne' === this.state.questionOneRadio }
```

Comment:
Not a big deal, but we don't generally do yoda conditionals in calypso. Since equality checks are always done with `===`, we're not overly worried about the accidental assignment that yoda conditionals are meant to avoid.